native/src/test/java/io/ayash/blockingqueue/BlockingQueueTest.java

```diff
@@ -200,82 +200,82 @@ public void testOfferWithTimeoutQueueGetsAvailable() throws InterruptedException
     @Test
     public void testAdd() {
         BlockingQueue queue = new BlockingQueue(2);
-        assertTrue(BlockingQueue.add(queue, 1));
-        assertTrue(BlockingQueue.add(queue, 2));
-        assertThrows(IllegalStateException.class, () -> BlockingQueue.add(queue, 3));  // This should throw an exception since the queue is full
+        assertTrue(queue.add( 1));
+        assertTrue(queue.add( 2));
+        assertThrows(IllegalStateException.class, () -> queue.add( 3));  // This should throw an exception since the queue is full
     }
 
     // ----------------------- Test contains ---------------------
     @Test
     public void testContains() throws InterruptedException {
         BlockingQueue queue = new BlockingQueue(5);
-        BlockingQueue.put(queue, 1);
-        BlockingQueue.put(queue, 2);
-        BlockingQueue.put(queue, 3);
-
-        assertTrue(BlockingQueue.contains(queue, 1));
-        assertTrue(BlockingQueue.contains(queue, 2));
-        assertTrue(BlockingQueue.contains(queue, 3));
-        assertFalse(BlockingQueue.contains(queue, 4));
+        queue.put( 1);
```

Comment:
```suggestion
        queue.put(1);
```
check and resolve these checkstyle errors in other places as well..